### PR TITLE
Added JRE installation instructions.

### PR DESCRIPTION
**Pull request by rasmi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,6 +91,18 @@ pipenv shell
 
 Finally, run `pytest`. If no tests fail, you are ready to develop!
 
+**NOTE**: If some `recidiviz/tests/ingest/aggregate` tests fail, you may need to install the Java Runtime Environment (JRE).
+
+On a Mac with [Homebrew](https://brew.sh/), you can install the JRE with:
```
Comment (jessex) [inline README.md]: Are there any version requirements to be aware of?

**Comment by rasmi** [comment]
I'm fine adding a line that says:

> Alternatively, you can ignore those tests with:
> ```
> pytest --ignore=recidiviz/tests/ingest/aggregate
> ```

Our volunteer scraper writers need not worry about aggregate ingest tests -- it may be a pain to install the JRE for some.

**Comment by jessex** [comment]
I think both of these instructions are worth mentioning. Let's be as thorough as possible.

File: README.md (hunk 1 above)
Comment (rasmi) [reply README.md]: According to Tabula docs, Java >= 7: https://github.com/tabulapdf/tabula/tree/cd8a1b03e1d499a6a480f7d38c1b1ba86967935c#using-tabula

If installing with `brew` or `apt`, you have to explicitly list a lower version (<=6) for these commands not to work, so the instructions should be fine, but I'm happy to add `JRE >=7` to be explicit!

**Pull request merged**